### ISSUE TITLE
[1.4.x] Calling module gives another twig error

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#748]
```
Unexpected token "operator" of value "not" ("end of statement block" expected) in @MUMUImageModule/Album/viewQuickNav.html.twig at line 14.
```

**Comment by Portugao** [comment]
There is a 'is' missing.

**Comment by Guite** [comment]
Fixed in 766e2e98808f1c0ff1f0db3af8cc7d28df6ac99a
